events/tests/test_event_get.py

```diff
@@ -94,14 +95,22 @@ def assert_event_fields_exist(data, version='v1'):
     assert_fields_exist(data, fields)
 
 
-def assert_events_in_response(events, response):
+def assert_events_in_response(events, response, query=''):
     response_event_ids = {event['id'] for event in response.data['data']}
     expected_event_ids = {event.id for event in events}
-    assert response_event_ids == expected_event_ids
+    if query:
+        assert response_event_ids == expected_event_ids, f'\nquery: {query}'
```

Comment:
the point of this change is to see the query in the pytest output, should assertion fail